test/test727.js

```diff
@@ -0,0 +1,18 @@
+if(typeof exports === 'object') {
+	var assert = require("assert");
+	var alasql = require('..');
+	var fs = require("fs");
+}
+
+var test = '727'; // insert test file number
+
+describe('Test '+test+' - INTO CSV', function() {
+	
+	after(function(){
+		fs.unlink('sdfsfd.csv');
+        });
+
+	it('With quote = \'\'', function(){
+		alasql("SELECT \"1\" INTO CSV('sdfsfd', {quote:''})");
+        });
+});

```

Comment:
There should always be an `assert(...)` in a test. I can see the point in only testing that it does not fail but would be good to read the file (with fs) and verify the content is as you would like it to be. 

What about if we have more than one row? What if we have a string with more words?

Would be good to see som tests verifying a few more scenarios.